src/apiflask/scaffold.py

```diff
@@ -311,7 +311,15 @@ def decorator(f):
                     _annotate(f, args=[])
                 # TODO: Support set example for request parameters
                 f._spec['args'].append((schema, location))
-            return use_args(schema, location=location, **kwargs)(f)
+
+            @wraps(f)
```

Comment:
Please move the code to a separate function to prevent repetition and call it for each decorator (see the [example implementation](https://github.com/miguelgrinberg/APIFairy/blob/main/src/apifairy/decorators.py#L24-L33) in APIFairy).